apps/verticalhome/js/stores/item.js

```diff
@@ -217,8 +218,10 @@
           self.gridOrder = configurator.getGrid();
           self.populate(cb);
         });
-        window.addEventListener('configuration-pin-app-ready', function onPinAppReady() {
-          window.removeEventListener('configuration-pin-app-ready', onPinAppReady);
+        window.addEventListener('configuration-pin-app-ready',
```

Comment:
@sergkashin how do you think, maybe it'll be better to change event name & function name to something shorter to fit 70 characters? instead of making this :crying_cat_face: indentation?